File: Web/Vaishnavi-Singh20002_VaishnaviSingh_2125ec1060_02/GettingStarted/isnumberprime.cpp
#include <iostream>
using namespace std;
int main(){
    int t,n;
    cin >> t;
    for(int i=1;i<=t;i++){
        cin>>n;
       int count = 0;
        for(int j = 2; j*j <= n; j++){
            if(n % j == 0){
                count++;
            }
        }
        if(count == 0){
            cout<<"prime\n";
        }
        else{
            cout<<"not prime\n";
        }
    }
    return 0;
}

File: Web/Macabre908_AbhishekDubey_2125cse1175_2/GettingStarted/isNumberPrime.java
import java.util.*;

public class isNumberPrime {
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        for(int x = 1; x <= t; x++){
            int n = sc.nextInt();
            int count = 1;
            for(int i = 1; i <= n / 2; i++){
                if(n % i == 0)
                count++;
            }

            if(count == 2)
            System.out.println("prime");
            else
            System.out.println("not prime");
        }

        sc.close();
    }
}
